.github/workflows/gradle.yml

```diff
@@ -20,6 +20,7 @@ jobs:
     strategy:
       matrix:
         java: ['8', '11', '15']
+        k8s: ['1_16', '1_17', '1_18', '1_19']
```

Comment:
How to make this work wrt ```
# WARNING: Do not edit this file directly. Instead, go to:
#
# https://github.com/micronaut-projects/micronaut-project-template/tree/master/.github/workflows
#
# and edit them there. Note that it will be sync'ed to all the Micronaut repos
```

Reply:
This is going to be problematic. Let's discuss in a call.